code/game/turfs/open.dm

```diff
@@ -172,7 +172,7 @@
 /turf/open/indestructible/grass
 	name = "grass patch"
 	desc = "Yep, it's grass."
-	icon_state = "grass"
+	icon_state = "grass1"
```

Comment:
why is this changed

Reply:
because "grass" doesn't exist anymore. it's grass1-4

Reply:
> because "grass" doesn't exist anymore. it's grass1-4

sorry this needs to be in it's own separate PR